### PR TITLE
okta method for refreshing tokens

### PR DESCRIPTION
**Pull request by clink-aaron**

### CODE REVIEW

**Comment by clink-aaron** [comment]
hi! I've tried to make this consistent with existing conventions from other packages, but if there's changes you'd like done I'd be glad to do whatever is needed to help move this along.

**Comment by clink-aaron** [comment]
@atymic bump

File: src/Okta/README.md [hunk 1]
```diff
@@ -84,6 +85,21 @@ public function logout(\Illuminate\Http\Request $request)
     return redirect($logoutUrl);
 }
 ```
+#### Refresh Token
+Using a refresh token allows an active user to maintain their session
```
Comment (atymic) [inline src/Okta/README.md:89]: Might be worth adding a docs link here?

**Review by atymic — APPROVED** [review]
LGTM one minor comment

File: src/Okta/README.md (hunk 1 above)
Comment (clink-aaron) [reply src/Okta/README.md]: @atymic clarified usage and added link to relevant docs.

**Review by atymic — APPROVED** [review]

**Pull request merged**